### DISCUSSION TITLE
Unable to see "🐪" using Konsole on Debian

### DISCUSSION THREAD

**Discussion by gianfelicevincenzo** [starship/starship#2093] [Q&A] [answered]
Hi. I've been testing spaceship recently and I couldn't help but notice how some symbols are not represented correctly..especially when I enter a folder with perl scripts on the prompt I get a "rectangle". The following is the code of the symbol in question:

```
00000000  f0 9f 90 aa 0a                                    |.....|
00000005
```

I installed all the "nerd fonts" fonts (more than 6gb of fonts!) In addition, the awesome fonts too, but nothing ... that symbol (and others) are not displayed correctly.

**Comment by matchai** [comment, marked as answer]
Hey Vincenzo. 👋
Please reopen this issue as a bug report so we can best troubleshoot the issue.
